pipenv/vendor/tomlkit/container.py

```diff
@@ -196,8 +196,7 @@ def remove(self, key):  # type: (Union[Key, str]) -> Container
             self._body[idx] = (None, Comment(Trivia(comment_ws="", comment=trivia.comment)))
         else:
             self._body[idx] = (None, Null())
-            super(Container, self).__delitem__(key.key)
-
+        super(Container, self).__delitem__(key.key)
```

Comment:
This diff is probably accidental?

Reply:
No, the key should be deleted unconditionally.